assets/messages.yml

```diff
@@ -88,6 +88,17 @@ messages:
 
         ~This issue has been temporarily closed as {color:#d04437}*Awaiting Response*{color}. Once the requested information has been delivered, the report will be reopened automatically.~
 
+        %quick_links%
+      fillname: []
+  attach-debug-screen-image:
+    - project: mc
+      name: Attach F3 debug screenshot
+      shortcut: fscreen
```

Comment:
`fscreen` does not make its meaning that obvious.
Maybe `f3screen` or `dscreen` (with "d" standing for "**d**ebug") would be better?

Reply:
I was not aware that numbers could be included in the shortcut. Changed to "f3screen"

Reply:
There already exists `shortcut: '297'` so I hope this should be fine (also for the browser extension).